src/App.tsx

```diff
@@ -11,18 +11,27 @@ import Recorder from './views/Recorder';
 
 // Hacky fix to redirect user to recorder page when recorder window is launched.
 // eslint-disable-next-line react/prop-types
-const Redirect: React.FC<Partial<RouteProps>> = ({ location }) => {
-  // eslint-disable-next-line react/prop-types
-  const path = new URLSearchParams(location?.search).get('redirect');
-  if (path === 'recorder') return <Recorder />;
-  return <MenuBar />;
-};
+// const Redirect2: React.FC<Partial<RouteProps>>  = (AltComponent: React.FC) => ({ location }) => {
+//   // eslint-disable-next-line react/prop-types
+//   const path = new URLSearchParams(location?.search).get('redirect');
+//   if (path === 'recorder') return <Recorder />;
+//   return <AltComponent />;
+// };
```

Comment:
I think we can remove this commented block of code out since it's no longer needed.